code/game/turfs/open/openspace.dm

```diff
@@ -14,8 +25,13 @@
 
 /turf/open/openspace/Initialize() // handle plane and layer here so that they don't cover other obs/turfs in Dream Maker
 	. = ..()
-	plane = FLOOR_OPENSPACE_PLANE
 	layer = OPENSPACE_LAYER
+	plane = OPENSPACE_PLANE
+
+#if DM_VERSION >= 513
+	vis_contents += new /atom/movable/openspace_backdrop()

```

Comment:
I think it should be done in another way, but I don't see it now.
Earlier, openspace turf played this role, but while objects from below inherit the openspace plane, we need something else that will play the role of darkness of depth.

One way to improve this is to create one openspace_backdrop for all open spaces. But what is the best way to do this, and will it even work?